docs/angular/api-angular/schematics/downgrade-module.md

```diff
@@ -6,7 +6,20 @@ Setup Downgrade Module
 
 ```bash
 ng generate downgrade-module ...
+```
+
+By default, Nx will search for `downgrade-module` in the default collection provisioned in `angular.json`.
+
+You can specify the collection explicitly as follows:
+
+```bash
+ng g @nrwl/angular:downgrade-module ...
+```
 
+Show what will be generated without writing to disk:
+
+```bash
+ng g null ... --dry-run
```

Comment:
There is still null here. This should be ng g downgrade-module or ng g @nrwl/angular:downgrade-module --dry-run